app/code/community/Bolt/Boltpay/Model/Observer.php

```diff
@@ -136,7 +136,15 @@ public function sendCompleteAuthorizeRequest($request)
      */
     public function sendOrderEmail($order)
     {
-        $order->sendNewOrderEmail();
+        try {
+            $order->sendNewOrderEmail();
+        } catch (Exception $e) {
+            // Catches errors that occur when sending order email confirmation (e.g. Mandrill API is down)
+            // and allows order creation to complete.
+            Mage::helper('boltpay/bugsnag')->notifyException($e);
```

Comment:
maybe wrap this with generic error message like `new Error('failed to send order email', $e)`?